src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationChartFeedbackPresenter.cs

```diff
@@ -112,11 +112,14 @@ protected void ConfigureColumnDimension(DataColumn column)
 
       protected abstract void AddCurvesFor(DataRepository repository, Action<DataColumn, Curve> action);
 
-      protected virtual void SelectedOutputChanged()
+      protected void SelectedOutputChanged()
       {
-         UpdateChartAxesScalings();
+         UpdateChartForSelectedOutput();
```

Comment:
This is now the abstract method. Once implementation specific is done, we refresh the chart display